plugins/modules/rds_cluster.py

```diff
@@ -187,11 +187,16 @@
         description:
           - Specifies the storage type to be associated with the DB cluster.
           - This setting is required to create a Multi-AZ DB cluster.
-          - When specified, a value for the I(iops) parameter is required.
-          - Defaults to C(io1).
+          - For multi-AZ DB clusters, I(storage_type) defaults to C(io1) and a value for the I(iops) parameter is required.
+          - For Aurora DB clusters, I(storage_type) defaults to C(aurora) standard.
+          - For mysql and postgres DB clusters, I(storage_type) defaults to C(io1).
+          - I(storage_type) require botocore >= 1.23.44.
+          - Support for aurora and aurora-iopt1 was added in release 7.6.0.
```

Comment:
We missed 7.6.0. I would suggest targeting 8.0.0 (that will be released next week) or 8.1.0 (if we miss 8.0.0).

Reply:
likely will miss 8.0.0, targeting 8.1.0